pkg/skaffold/sync/sync_test.go

```diff
@@ -921,58 +921,64 @@ var pod = &v1.Pod{
 }
 
 func TestPerform(t *testing.T) {
-	tests := []struct {
-		description string
```

Comment:
nit: I would revert and keep description as a field instead of making this a map. It is easier to read the tests that way and is the general best practice.

Reply:
done